tools/format.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -eu
```

Comment:
use long option names for readability:
```suggestion
set -o errexit
set -o nounset
```